### ISSUE TITLE
verbesserte Usability bei Abhängikeiten und Meldung "don't install..."

### ISSUE THREAD

**Issue by zonky2** [contao/contao-manager#542]
verbesserte Usability bei Abhängikeiten und Meldung "don't install..."

Das Thema wurde auf Slack besprochen https://contao.slack.com/archives/CJT61P8R1/p1586246884061000

Bei Fehlermeldungen wie hier beschrieben https://community.contao.org/de/showthread.php?77635-Installationsprobleme-mit-Contao-4-9-Managed-Edition kommen (unbedarfte) User nicht weiter. Es gibt immer wiederkehrende Erklärungen, welche Einstellungen am CM vorgenommen werden müssen, damit die Installation bzw. Auflösung durch läuft.

Hier ist die Frage, wie das verbessert werden kann - Vorschläge:
Netzmacht:
Zwei Optimierungswünsche:

1. Man kann auch nach dem Auswählen eines neuen Paketes auf Pakete aktualisieren klicken
2.a) Scheitert die Installation schlägt der CM vor es mit der Aktualisierung aller Pakete zu probieren (für den Fall, dass sie vorher nicht ausgewählt waren. )
2.b) Scheitert die Installation versucht der CM automatisch ein Update aller Pakete (bearbeitet) 

zonky2: FAQ Button mit Link zu https://docs.contao.org/manual/de/installation/contao-manager/#haeufige-fragen-zum-contao-manager und Beschreibung zur Problemlösung

**Comment by fritzmg** [comment]
I also think that, in case of any errors (or just some identifiable errors?), the Contao Manager should offer the possibility to do a full `composer update` (including the latest changes, i.e. when you added a package, that would require a full composer update to be installed).

**Comment by fiedsch** [comment]
Couldn't that (the full update) also be the default path as it is what most people (presumably) want anyway. (hope, I don't misunderstand the problem as I don't use the manager).

**Comment by fritzmg** [comment]
No, this would also update any other package.

**Comment by fiedsch** [comment]
Yes, of course. But why would I regularly not want this. I was talking about a possible default---the option to only update a single package could also be there, but I fail to see why it has to be the default (which might be my lack of understanding 😨).

**Comment by fritzmg** [comment]
If I install a package via the Contao Manager, I don't expect any other package to be updated. Otherwise it would lead to unforeseen consequences.

**Comment by fiedsch** [comment]
OK, so I always did it wrong and was lucky so far. Thanks @fritzmg

**Comment by fritzmg** [comment]
No, you did not do it wrong. If you wanted to update all packages, that obviously fine ;).

**Comment by fiedsch** [comment]
Most of the times I just wanted to add a package but did not mind if that updated all packages as a side effect (bc. updating all packages is what I would have done anyway at some later point).  That was the base of my reasoning.

**Comment by aschempp** [comment]
🤔 maybe we should offer an option to update all packages when installing one? Something like a dropdown (/dropup) on the "Apply" action?
